scm-record/src/ui.rs

```diff
@@ -3095,7 +3090,14 @@ impl Component for SectionView<'_> {
                 };
                 let toggle_box_rect = viewport.draw_component(x, y, &toggle_box);
                 let x = x + toggle_box_rect.width.unwrap_isize() + 1;
-                let text = format!("File mode changed from {before} to {after}");
+
+                let text = match mode {
+                    // TODO: It would be nice to render this as 'file was created with mode x' but we don't have access
+                    // to the file's mode to see if it was absent before here.
+                    FileMode::Rwx(mode) => format!("File mode changed to {mode:o}"),
```

Comment:
How about "File mode *set* to ..."? I think that sounds better if the file didn't exist before

Reply:
I'll see where I get to with https://github.com/arxanas/scm-record/pull/93#discussion_r1985673810 but if it doesn't work out then definitely agree

Reply:
As I couldn't figure that out, I've changed this to "set to...`

Reply:
As I couldn't figure that out, I've changed this to "set to..."